aws/cf_template.go

```diff
@@ -473,6 +473,7 @@ func newTargetGroup(spec *stackSpec, targetPortParameter string) *cloudformation
 	// custom target group healthcheck only supported when the target group protocol is != TCP
 	if protocol != "TCP" {
 		targetGroup.HealthCheckTimeoutSeconds = cloudformation.Ref(parameterTargetGroupHealthCheckTimeoutParameter).Integer()
+		targetGroup.HealthyThresholdCount = cloudformation.Integer(int64(spec.albHealthyThresholdCount))
```

Comment:
For some reason all other variables are referenced as template parameters. I am not sure yet why but I think we should follow the suit here.

Reply:
@szuecs @mikkeloscar Do you remember why do we use parameter refs elsewhere?

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html

Reply:
we discourage using parameter when the template is rendered by a dynamic language already like this project is, as it has a couple of downsides (such as no CFN diff), but I'm happy to adjust to comply whatever is desired

Reply:
I have no opinion on that.

Reply:
I think we should use parameters for consistency but I have opened an issue to reconsider https://github.com/zalando-incubator/kube-ingress-aws-controller/issues/454

Reply:
OTOH I see we are not using parameters for everything so maybe we should not bother introducing more now and it is ok as it is.